core/src/apps/bitcoin/sign_tx/__init__.py

```diff
@@ -70,7 +71,13 @@ async def sign_tx(
         if coin.decred:
             signer_class = decred.Decred
         elif coin.overwintered:
-            signer_class = zcash.Zcashlike
+            if (
+                coin.coin_name in ["Zcash", "Zcash Testnet"] and
+                msg.version == 5  # remove this condition after Zcash NU5 mainnet activation
+            ):
+                signer_class = zcash.signer.Zcash
+            else:
+                signer_class = zcashlike.Zcashlike
```

Comment:
I take it that `zcashlike.py` is for the Komodo and Koto coins. Are these going to keep using the older transaction format? If so, then I think we should be calling this `zcash_legacy.py` or `zcash_v4.py`, because "zcashlike" would seem to be implementing a superset of Zcash functionality, which it will no longer be doing.

Reply:
https://github.com/trezor/trezor-firmware/pull/2166/commits/71c85f9a8fe9a8a00a3ea779af87b71b1c384ed3

Reply:
Also the class name `Zcashlike` -> `ZcashV4`.

Reply:
https://github.com/trezor/trezor-firmware/pull/2166/commits/7c467f6fd3372d8c184f8b0a3ab18dac1e6e62cc